src/Breadcrumb/BreadcrumbLink.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+export default function BreadcrumbLink({ as, clickHandler, linkProps }) {
+  const {
+    label,
+    url,
+    className,
+    ...props
+  } = linkProps;
+  const addtlProps = {};
+
+  if (as === 'a' && url) {
```

Comment:
previously consumers would describe links as
```jsx
{ label: 'my link', url: '/my-link' }
```
which can't be mapped to `a`'s props, because it expects `href`, not `url`